src/unix/bsd-ifaddrs.c

```diff
@@ -108,6 +108,14 @@ int uv_interface_addresses(uv_interface_address_t** addresses, int* count) {
       address->netmask.netmask4 = *((struct sockaddr_in*) ent->ifa_netmask);
     }
 
+    if (ent->ifa_flags & IFF_BROADCAST && ent->ifa_broadaddr != NULL) {
+      if (ent->ifa_broadaddr->sa_family == AF_INET) {
+        address->broadcast.broadcast4 = *((struct sockaddr_in*) ent->ifa_broadaddr);
+      }
+    } else {
+      memset(&address->broadcast.broadcast6, 0, sizeof(struct sockaddr_in6));
```

Comment:
I suppose this already probably usually set all of `address->broadcast.broadcast4` to zero also, but perhaps it should be explicit?
```suggestion
      memset(&address->broadcast, 0, sizeof(address->broadcast));
```

Reply:
It completely zeros the union, so individually clearing them would be duplicative.

Reply:
Resolved when `broadcast6` went away...